src/components/label/label.scss

```diff
@@ -14,14 +14,14 @@ ion-label {
   font-size: inherit;
   text-overflow: ellipsis;
   white-space: nowrap;
+
+  pointer-events: none;
```

Comment:
Why move pointer-events to just ion-label here? What if they used it outside of an input, and have a link inside of it?